install-protoc.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Installs protoc; used for generating JS protobuf stubs.
+# This script is intended to be run by CI
+
+set -ex
+
+if [[ -z "$PROTOC_VER" ]]; then
+  echo "PROTOC_VER environment variable not set"
+  exit 1
+fi
+
+curl -sSL \
+  https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VER}/protoc-${PROTOC_VER}-linux-$(uname -m).zip \
```

Comment:
Note that this doesn't use `uname -s` since protoc uses lowercase names. I figured since this is for CI it doesn't matter, but I can make probably do a lowercase transformation somehow if necessary.